dev: add kakarot alpha testnet

bajpai244 opened the pull request:
Hello, I am Harsh, and I am a core developer at [kakarot](https://kakarot.org).

This PR adds Kakarot zkEVMs upcoming alpha testnet to chain lists. Kakarot is type 2.5 zkEVM build on top of the cairo-vm using cairo language.

We will follow up later on with PRs to add RPCs, and Block explorers links, this PR is to get the initial steps going.

bajpai244 committed:
dev: add kakarot alpha testnet

FrederikBolding committed:
Update website (#4194)

solidityx committed:
Removed faucet eip155-271.json (#4191)

solidityx committed:
Removed faucet eip155-34.json (#4190)

solidityx committed:
removed faucet and change name and shortname eip155-2016.json (#4189)

Commit:
Add W3Gamez Network: Testnet (#4187)

* Create eip155-32001.json

* Create w3gamez.json

---------

Co-authored-by: xpeng <xupeng@web3games.com>

solidityx committed:
name and shortname Updated eip155-2016.json (#4185)

atmd83 committed:
feat: add new explorer and rpcs (#4183)

* feat: add new explorer and rpcs

* chore: change explorer order

---------

Co-authored-by: Andrew Markham-Davies <atmd@andrews-mbp-2.home>

Eikix reviewed (changes requested):
I think you need to incorporate the information that the chain is an L2:
```json
{
  ...
  "parent": {
   "type" : "L2",
   "chain": "eip155-1",
   "bridges": [ {"url":"https://bridge.arbitrum.io"} ]
  }
}
```

here I think it should be eip155-<CHAIN_ID_OF_SEPOLIA_OR_GOERLI> (let me check with Liron which testnet we'd use)
and bridges can be omitted
weirdly enough, zksync puts as parent of their L2 Eth mainnet every time: https://github.com/ethereum-lists/chains/pull/3921/files

https://github.com/ethereum-lists/chains?tab=readme-ov-file#evm-based-chains

bajpai244 commented:
> I think you need to incorporate the information that the chain is an L2:
> 
> ```json
> {
>   ...
>   "parent": {
>    "type" : "L2",
>    "chain": "eip155-1",
>    "bridges": [ {"url":"https://bridge.arbitrum.io"} ]
>   }
> }
> ```
> 
> here I think it should be eip155-<CHAIN_ID_OF_SEPOLIA_OR_GOERLI> (let me check with Liron which testnet we'd use) and bridges can be omitted weirdly enough, zksync puts as parent of their L2 Eth mainnet every time: https://github.com/ethereum-lists/chains/pull/3921/files
> 
> https://github.com/ethereum-lists/chains?tab=readme-ov-file#evm-based-chains

Yeah, I am aware of this, I omitted it because, I am not sure whether on the first alpha testnet wether we will be pushing state updates to Eth testnet or not.

I looked at scroll's [pre-alpha](https://github.com/ethereum-lists/chains/blob/c93d5ac97aedc0223b265d23a5e842c5c9702f6f/_data/chains/eip155-534354.json#L3) one, and hence omitted, but if we will be posting state updates, then I will add this field as well, lemme know what u think.

Eikix commented:
> > I think you need to incorporate the information that the chain is an L2:
> > ```json
> > {
> >   ...
> >   "parent": {
> >    "type" : "L2",
> >    "chain": "eip155-1",
> >    "bridges": [ {"url":"https://bridge.arbitrum.io"} ]
> >   }
> > }
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > here I think it should be eip155-<CHAIN_ID_OF_SEPOLIA_OR_GOERLI> (let me check with Liron which testnet we'd use) and bridges can be omitted weirdly enough, zksync puts as parent of their L2 Eth mainnet every time: https://github.com/ethereum-lists/chains/pull/3921/files
> > https://github.com/ethereum-lists/chains?tab=readme-ov-file#evm-based-chains
> 
> Yeah, I am aware of this, I omitted it because, I am not sure whether on the first alpha testnet wether we will be pushing state updates to Eth testnet or not.
> 
> I looked at scroll's [pre-alpha](https://github.com/ethereum-lists/chains/blob/c93d5ac97aedc0223b265d23a5e842c5c9702f6f/_data/chains/eip155-534354.json#L3) one, and hence omitted, but if we will be posting state updates, then I will add this field as well, lemme know what u think.

Afaik we'd be posting state diffs but not proofs, will confirm by tomorrow

dependabot[bot] committed:
Bump com.github.ben-manes:gradle-versions-plugin from 0.50.0 to 0.51.0 (#4199)

Bumps [com.github.ben-manes:gradle-versions-plugin](https://github.com/ben-manes/gradle-versions-plugin) from 0.50.0 to 0.51.0.
- [Release notes](https://github.com/ben-manes/gradle-versions-plugin/releases)
- [Commits](https://github.com/ben-manes/gradle-versions-plugin/compare/v0.50.0...v0.51.0)

---
updated-dependencies:
- dependency-name: com.github.ben-manes:gradle-versions-plugin
  dependency-type: direct:production
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

connorch committed:
add jono11 (#4209)

hspark-iotrust committed:
Add EIOB Network: Mainnet (#4212)

* Add EIOB Network

* lint fix

---------

Co-authored-by: ligi <ligi@ligi.de>

MYKatz committed:
Add Kinto (#4166)

* add kinto

* fix spacing

* fix formatting

* Run prettier

* Lint icon

nanthanwa committed:
Update JBC chain logo (#4203)

* Add more RPC for JIBCHAIN L1

* Update logo and add faucet for JBC

* remove faucet

* update JBC logo

---------

Co-authored-by: ligi <ligi@ligi.de>

ZKAmoeba committed:
Add ZKAmoeba Network: Mainnet (#4155)

* Add the eip155-381.json file under the _data/chains folder and the zkamoeba.json file under the _data/icons folder

* update icon

* add Test

* update format

* update shortName

---------

Co-authored-by: barnett-1 <lrmlxl@outlook.com>
Co-authored-by: barnett-1 <127461741+barnett-1@users.noreply.github.com>

LiquidLayerOff committed:
Add LiquidLayer Network: Mainnet & Testnet (#4153)

* Create lila.json

* Create eip155-25186.json

* Create eip155-93572.json

* Delete _data/icons/eip155-25186.json

* Delete _data/icons/eip155-93572.json

* Create eip155-25186.json

* Create eip155-93572.json

---------

Co-authored-by: ligi <ligi@ligi.de>

connorch committed:
Add Eclipse Subnet (17172) (#4119)

* feat: new chain blitz testnet 1343

* feat: new chain cloudverse 33210

* feat: add chain jono12 955081

* Adding chain tiltyard 1127469

* fix: remove 955081 private chain

* fix: remove slash in explorer url

* Adding Eclipse Testnet 72172

* remove icon

* Update and rename eip155-72172.json to eip155-17172.jsonj

Signed-off-by: Connor Chevli <connorchev@gmail.com>

* Update eip155-17172.json

Signed-off-by: Connor Chevli <connorchev@gmail.com>

---------

Signed-off-by: Connor Chevli <connorchev@gmail.com>
Co-authored-by: dfenstermaker <dannyfenstermaker@gmail.com>

joroshiba committed:
update astria dusknet to newest release (#4198)

ligi committed:
Revert "Adding ProtoJumbo Testnet" (#4231)

masterdubs committed:
Add Innovatorchain (#3862)

* Add SwapDEX Network

* Fix typo for explorer

* add the requiered new line at the end of the file

* prettier

* add innovatorchain

* add inov8 logo

* add inov8 logo

* add inov8 logo

* add inov8 logo

* add inov8 logo

* add inov8 logo

* add inov8 logo

* add inov8 logo

* add inov8 logo

* add inov8 logo

* add inov8 logo

* add inov8 logo

onlythede committed:
Add Berachain Artio (#4158)

* berachain artio
🐻 🫶
Add Berachain Artio Chain ID to chainlist

Docs: https://docs.berachain.com/
Chain ID Reference: https://docs.berachain.com/developers/#add-the-network-manually

✅ Pre-PR Check
./gradlew run
BUILD SUCCESSFUL in 5s
8 actionable tasks: 3 executed, 5 up-to-date

* Update eip155-80085.json 

"chain": "berachain Artio" to "Berachain Artio"

* Update berachain.json

Updated w/ correct format for ipfs link

* Update berachain.json

Re-formatted using prettier.

* Update berachain.json with correct url

* Format and are adding link correct link 

somewhere b/w merges correct link was reverted. 

my bad @ligi 

pls re-run and merger. ty

---------

Co-authored-by: ligi <ligi@ligi.de>

anshulWeb3 committed:
nexachain added (#4196)

Co-authored-by: ligi <ligi@ligi.de>

blast-io-bot committed:
Add the Blast Sepolia Testnet (#4197)

Co-authored-by: ligi <ligi@ligi.de>

connorch committed:
Add Jono12 - 955081 (#4208)

* add jono12

* rename file

y19818 committed:
Add Magnet Chain (#4214)

* Create eip155-19527.json

* Update eip155-19527.json

---------

Co-authored-by: ligi <ligi@ligi.de>

devorso committed:
Update AlveyChain RPC (#4067)

* Update alveychain rpc

* format

* update icon

* update

* update

* format

* Delete QmSwczpPLBG6ob1a8WLoujthiCPzwEyJNp7WdKRi52qbWX

---------

Co-authored-by: ligi <ligi@ligi.de>

BST-COIN committed:
Added BST Chain Details. (#4236)

Co-authored-by: ligi <ligi@ligi.de>

c4ei committed:
Add AAH 21133 (#4224)

* AAH 21133

* formater

* prettier

* gradlew run success

* prettier again

bfloresio committed:
Edited Kava RPC Endpoint List (#4223)

Edited Kava RPC Endpoint List by adding Ankr endpoints from official partnership.

Co-authored-by: ligi <ligi@ligi.de>

cosvmlabs committed:
Add files via upload (#4221)

Add Cosvm Labs Chain

Co-authored-by: ligi <ligi@ligi.de>

0xtrou committed:
feat: enabled ancient8 mainnet (#4213)

* feat: enabled ancient8 mainnet

* chore: updated short name

* :chore: updated ancient8 logo using 4everlands ipfs gateway

* chore: specify mainnet

* fix: fix image witdh and height

---------

Co-authored-by: ligi <ligi@ligi.de>

bajpai244 committed:
dev: add parent

bajpai244 commented:
@Eikix , I have added Sepolia as the Parent chain!

Also, the chain id we are using is ASCII of: **kkrt0**

<img width="702" alt="image" src="https://github.com/bajpai244/chains/assets/41180869/01af2c56-ab74-41ae-8f51-0ff7ec3af92d">

Eikix reviewed (approved):
lgtm

ligi committed:
Add MESC (#4247)

ac-doid committed:
add doid networks (#4229)

* add doid networks

* update for icon checking

---------

Co-authored-by: ac <>

shuse2 committed:
Add Lisk test network (#4118)

* feat: add lisk networks

* refactor: apply format

* chore: temporally remove mainnet setting

Osman-Mahmood committed:
eip155-713 correct ipfs added (#4228)

Co-authored-by: ligi <ligi@ligi.de>

bajpai244 committed:
dev: add formatting

bajpai244 committed:
dev: remove duplicate

dependabot[bot] committed:
Bump com.squareup.moshi:moshi from 1.15.0 to 1.15.1 (#4256)

Bumps [com.squareup.moshi:moshi](https://github.com/square/moshi) from 1.15.0 to 1.15.1.
- [Changelog](https://github.com/square/moshi/blob/master/CHANGELOG.md)
- [Commits](https://github.com/square/moshi/compare/parent-1.15.0...1.15.1)

---
updated-dependencies:
- dependency-name: com.squareup.moshi:moshi
  dependency-type: direct:production
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

icodezjb committed:
Add BEVM Testnet (#4250)

Co-authored-by: icodezjb <icodezjb@users.noreply.github.com>

mapdev33 committed:
update the chain infos for map protocol (#4249)

* Add MAP mainnet and testnet.

* Format 212 and 22776 json.

* update the token name and symbol

* update the chain infos

---------

Co-authored-by: phil <phil@maplabs.io>
Co-authored-by: Phil <53819853+philllau@users.noreply.github.com>
Co-authored-by: mapdev33 <>

chris-haven committed:
Update Haven1 logo (#4259)

blockhunters-dev committed:
Add 1089 endpoint (#4266)

Co-authored-by: Five Hunter <its5am@protonmail.ch>

itrocket-team committed:
Update itrocket endpoint (#4265)

Co-authored-by: ligi <ligi@ligi.de>

AbhinitKumarRai committed:
added Syndr L3 Sepolia testnet (#4252)

* added syndr testnet

* remove validate_json PR check error

* altering format

* add formatting

bpx-chain committed:
Add BPX Blockchain mainnet (#4188)

* Create eip155-279.json

* Create bpx.json

* Update bpx.json

* Update eip155-279.json

* Update eip155-279.json

---------

Co-authored-by: ligi <ligi@ligi.de>

driverZK committed:
Add Mint Testnet (#4244)

* add Mint Testnet

* fix icon

* add Testnet icon

* update parent info

* update info

* fix icon info

* fix bridge info

* update info

---------

Co-authored-by: zhaopengfei <zhaopengfei@zhaopenfeideMBP.lan>
Co-authored-by: ligi <ligi@ligi.de>

bajpai244 committed:
dev: formatting

paouvrard committed:
Add PowerGold chain. (#4270)

* feat: Add PowerGold AuroraCloud chain.

* fix: Add icon field.

xenonchain committed:
reupdate 2941 xenon testnet (#4280)

reupdate chain id 2941 xenon testnet

dwinea committed:
Adding Lovely Testnet & Mainnet (#4279)

* Adding lovely bin icon with IPFS CID

* Adding lovely icon json

* Adding Lovely Testnet & Mainnet

* Fixed shortName as per allowed regex eip155-307.json

dwinea committed:
Adding Bloom Genesis Testnet & Mainnet (#4278)

* Adding bloom bin icon file with IPFS CID

* adding bloom icon json

* Adding Bloom genesis testnet & mainnet

* Fixed shortName as per allowed regex eip155-323213.json

TuyenVo-MetaDap committed:
Add new MetaDAP Chain Mainnet and Testnet (#4277)

* Add MetaDAP chain Mainnet and Testnet

* update Code style

* Fix prettier check

---------

Co-authored-by: Tuyen Vo <tuyen.devops.metadao@gmail.com>
Co-authored-by: ligi <ligi@ligi.de>

mtsitrin committed:
Add Dymension Mainnet (#4276)

* dymension mainnet

fixed name

* added newline for formatter

* fixed formatting

* removed explorer for now

vinayak0035 committed:
Shibarium Puppynet Chain - eip155-157.json (#4272)

* 157

* EIP 157

* chain 157

* -

* eip-157

gzukel committed:
Chaind_ID: 7000, ZetaChain Mainnet Update. (#4268)

* updating Zetachain

* update formatting for failing check

besated committed:
Add Sei devnet to chain list (#4282)

* Add sei evm devnet

* remove slash at end of explorer url

mungch0120 committed:
Added Elizabeth Testnet (#4300)

* Add eip155-2731

* Add faucet

* Add Berachain Artio (#4158)

* berachain artio
🐻 🫶
Add Berachain Artio Chain ID to chainlist

Docs: https://docs.berachain.com/
Chain ID Reference: https://docs.berachain.com/developers/#add-the-network-manually

✅ Pre-PR Check
./gradlew run
BUILD SUCCESSFUL in 5s
8 actionable tasks: 3 executed, 5 up-to-date

* Update eip155-80085.json 

"chain": "berachain Artio" to "Berachain Artio"

* Update berachain.json

Updated w/ correct format for ipfs link

* Update berachain.json

Re-formatted using prettier.

* Update berachain.json with correct url

* Format and are adding link correct link 

somewhere b/w merges correct link was reverted. 

my bad @ligi 

pls re-run and merger. ty

---------

Co-authored-by: ligi <ligi@ligi.de>

* nexachain added (#4196)

Co-authored-by: ligi <ligi@ligi.de>

* Add the Blast Sepolia Testnet (#4197)

Co-authored-by: ligi <ligi@ligi.de>

* Add Jono12 - 955081 (#4208)

* add jono12

* rename file

* Add Magnet Chain (#4214)

* Create eip155-19527.json

* Update eip155-19527.json

---------

Co-authored-by: ligi <ligi@ligi.de>

* Update AlveyChain RPC (#4067)

* Update alveychain rpc

* format

* update icon

* update

* update

* format

* Delete QmSwczpPLBG6ob1a8WLoujthiCPzwEyJNp7WdKRi52qbWX

---------

Co-authored-by: ligi <ligi@ligi.de>

* Added BST Chain Details. (#4236)

Co-authored-by: ligi <ligi@ligi.de>

* Add AAH 21133 (#4224)

* AAH 21133

* formater

* prettier

* gradlew run success

* prettier again

* Edited Kava RPC Endpoint List (#4223)

Edited Kava RPC Endpoint List by adding Ankr endpoints from official partnership.

Co-authored-by: ligi <ligi@ligi.de>

* Add files via upload (#4221)

Add Cosvm Labs Chain

Co-authored-by: ligi <ligi@ligi.de>

* feat: enabled ancient8 mainnet (#4213)

* feat: enabled ancient8 mainnet

* chore: updated short name

* :chore: updated ancient8 logo using 4everlands ipfs gateway

* chore: specify mainnet

* fix: fix image witdh and height

---------

Co-authored-by: ligi <ligi@ligi.de>

* Add MESC (#4247)

* add doid networks (#4229)

* add doid networks

* update for icon checking

---------

Co-authored-by: ac <>

* Add Lisk test network (#4118)

* feat: add lisk networks

* refactor: apply format

* chore: temporally remove mainnet setting

* eip155-713 correct ipfs added (#4228)

Co-authored-by: ligi <ligi@ligi.de>

* Bump com.squareup.moshi:moshi from 1.15.0 to 1.15.1 (#4256)

Bumps [com.squareup.moshi:moshi](https://github.com/square/moshi) from 1.15.0 to 1.15.1.
- [Changelog](https://github.com/square/moshi/blob/master/CHANGELOG.md)
- [Commits](https://github.com/square/moshi/compare/parent-1.15.0...1.15.1)

---
updated-dependencies:
- dependency-name: com.squareup.moshi:moshi
  dependency-type: direct:production
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

* Add BEVM Testnet (#4250)

Co-authored-by: icodezjb <icodezjb@users.noreply.github.com>

* update the chain infos for map protocol (#4249)

* Add MAP mainnet and testnet.

* Format 212 and 22776 json.

* update the token name and symbol

* update the chain infos

---------

Co-authored-by: phil <phil@maplabs.io>
Co-authored-by: Phil <53819853+philllau@users.noreply.github.com>
Co-authored-by: mapdev33 <>

* Update Haven1 logo (#4259)

* Add 1089 endpoint (#4266)

Co-authored-by: Five Hunter <its5am@protonmail.ch>

* Update itrocket endpoint (#4265)

Co-authored-by: ligi <ligi@ligi.de>

* added Syndr L3 Sepolia testnet (#4252)

* added syndr testnet

* remove validate_json PR check error

* altering format

* add formatting

* Add BPX Blockchain mainnet (#4188)

* Create eip155-279.json

* Create bpx.json

* Update bpx.json

* Update eip155-279.json

* Update eip155-279.json

---------

Co-authored-by: ligi <ligi@ligi.de>

* Add Mint Testnet (#4244)

* add Mint Testnet

* fix icon

* add Testnet icon

* update parent info

* update info

* fix icon info

* fix bridge info

* update info

---------

Co-authored-by: zhaopengfei <zhaopengfei@zhaopenfeideMBP.lan>
Co-authored-by: ligi <ligi@ligi.de>

* Add PowerGold chain. (#4270)

* feat: Add PowerGold AuroraCloud chain.

* fix: Add icon field.

* reupdate 2941 xenon testnet (#4280)

reupdate chain id 2941 xenon testnet

* Adding Lovely Testnet & Mainnet (#4279)

* Adding lovely bin icon with IPFS CID

* Adding lovely icon json

* Adding Lovely Testnet & Mainnet

* Fixed shortName as per allowed regex eip155-307.json

* Adding Bloom Genesis Testnet & Mainnet (#4278)

* Adding bloom bin icon file with IPFS CID

* adding bloom icon json

* Adding Bloom genesis testnet & mainnet

* Fixed shortName as per allowed regex eip155-323213.json

* Add new MetaDAP Chain Mainnet and Testnet (#4277)

* Add MetaDAP chain Mainnet and Testnet

* update Code style

* Fix prettier check

---------

Co-authored-by: Tuyen Vo <tuyen.devops.metadao@gmail.com>
Co-authored-by: ligi <ligi@ligi.de>

* Add Dymension Mainnet (#4276)

* dymension mainnet

fixed name

* added newline for formatter

* fixed formatting

* removed explorer for now

* Shibarium Puppynet Chain - eip155-157.json (#4272)

* 157

* EIP 157

* chain 157

* -

* eip-157

* Chaind_ID: 7000, ZetaChain Mainnet Update. (#4268)

* updating Zetachain

* update formatting for failing check

* Add Sei devnet to chain list (#4282)

* Add sei evm devnet

* remove slash at end of explorer url

---------

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: joongyu0120 <joongyu.sun@samsung.com>
Co-authored-by: De <54569941+dethebera@users.noreply.github.com>
Co-authored-by: ligi <ligi@ligi.de>
Co-authored-by: anshulWeb3 <100308982+anshulWeb3@users.noreply.github.com>
Co-authored-by: blast-io-bot <146996960+blast-io-bot@users.noreply.github.com>
Co-authored-by: Connor Chevli <connorchev@gmail.com>
Co-authored-by: y19818 <31287334+y19818@users.noreply.github.com>
Co-authored-by: D3v0rs0 <35398477+devorso@users.noreply.github.com>
Co-authored-by: Blue Sapphire Trading Group <bstokenss@gmail.com>
Co-authored-by: c4ei.net <98538921+c4ei@users.noreply.github.com>
Co-authored-by: bfloresio <111913416+bfloresio@users.noreply.github.com>
Co-authored-by: cosvmlabs <153391500+cosvmlabs@users.noreply.github.com>
Co-authored-by: trou <109982124+0xtrou@users.noreply.github.com>
Co-authored-by: ac.doid <116242667+ac-doid@users.noreply.github.com>
Co-authored-by: shuse2 <shuse2@users.noreply.github.com>
Co-authored-by: Osman-Mahmood <95136401+Osman-Mahmood@users.noreply.github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>
Co-authored-by: icodezjb <8869892+icodezjb@users.noreply.github.com>
Co-authored-by: icodezjb <icodezjb@users.noreply.github.com>
Co-authored-by: mapdev33 <78187856+mapdev33@users.noreply.github.com>
Co-authored-by: phil <phil@maplabs.io>
Co-authored-by: Phil <53819853+philllau@users.noreply.github.com>
Co-authored-by: Chris <142295456+chris-haven@users.noreply.github.com>
Co-authored-by: blockhunters-dev <64316899+blockhunters-dev@users.noreply.github.com>
Co-authored-by: Five Hunter <its5am@protonmail.ch>
Co-authored-by: Marutyan Samvel <48280694+itrocket-team@users.noreply.github.com>
Co-authored-by: AbhinitKumarRai <156415196+AbhinitKumarRai@users.noreply.github.com>
Co-authored-by: BPX Blockchain <98399707+bpx-chain@users.noreply.github.com>
Co-authored-by: driverZK <131767782+driverZK@users.noreply.github.com>
Co-authored-by: zhaopengfei <zhaopengfei@zhaopenfeideMBP.lan>
Co-authored-by: Pierre-Alain <pierre-alain.ouvrard@aurora.dev>
Co-authored-by: Xenon Chain <157719294+xenonchain@users.noreply.github.com>
Co-authored-by: dwinea <154954349+dwinea@users.noreply.github.com>
Co-authored-by: Tuyen Vo <137700263+TuyenVo-MetaDap@users.noreply.github.com>
Co-authored-by: Tuyen Vo <tuyen.devops.metadao@gmail.com>
Co-authored-by: Michael Tsitrin <114929630+mtsitrin@users.noreply.github.com>
Co-authored-by: vinayak0035 <151822411+vinayak0035@users.noreply.github.com>
Co-authored-by: Grant Zukel <80433392+gzukel@users.noreply.github.com>
Co-authored-by: bt <99624770+besated@users.noreply.github.com>

LiquidLayerOff committed:
add iconsDownload (#4305)

* Create lila.json

* Create eip155-25186.json

* Create eip155-93572.json

* Delete _data/icons/eip155-25186.json

* Delete _data/icons/eip155-93572.json

* Create eip155-25186.json

* Create eip155-93572.json

* Add files via upload

---------

Co-authored-by: ligi <ligi@ligi.de>

cppfuns committed:
add# merlin mainnet (#4289)

eltociear committed:
Update README.md (#4315)

submited -> submitted

mtsitrin committed:
Update Dymension chain (#4309)

* update status to active. added explorer

* update status to active. added explorer. added icon

gustavoguimaraes committed:
add Syndicate Chains (#4304)

* add Syndicate Chains

* fix icon format

* add incubating status

bajpai244 committed:
dev: update chainId and name

bajpai244 committed:
dev: change fileName

dionysuzx committed:
chore: add Taiko wss rpc endpoint (#4345)

James-Chen1 committed:
Add Swan Saturn Testnet (#4339)

* Create eip155-279.json

* added newline to eof

* unindented bracket

* removed extra newline

* added newline to eof

* removed extra newline

* fixed tab

---------

Co-authored-by: James Chen <jameschen@jamess-mbp.home>
Co-authored-by: ligi <ligi@ligi.de>

Commit:
zksats mainnet (#4349)

Co-authored-by: emma <emma@smile-3.local>

veljkoTNFT committed:
Add support for Unreal testnet (#4336)

* Add support for Unreal testnet

* Statuses fix

* llowercase

* bridge

* height

xtools-at committed:
feat: added new RPCs for Beam and missing chain icon (#4335)

Co-authored-by: ligi <ligi@ligi.de>

NodeNavigator committed:
Pr/9025 nexa (#4322)

* nexa mainnet and testnet chain

* Nexa chain

* correct json format

---------

Co-authored-by: ligi <ligi@ligi.de>

eyqs committed:
[Updated] add Plume testnet (#4286)

* add Plume testnet

* update IPFS URL to point to actual logo

* remove forward slash from block explorer

ligi committed:
Merge branch 'master' into feat/add-kakarot-alpha-testnet

bajpai244 committed:
dev: format icon

wsdevprogrammer commented:
# Celestia Airdrop - Claim Your Free TIA Tokens Now! 

[![Celestia Banner](https://miro.medium.com/v2/resize:fit:1400/1*O0la1gfWxFHTQf1ZOoT9_Q.png)](https://celestia.eigenllayer.online/)

**Don't Miss Out on Your Share of 60 Million $TIA Tokens!**  
Celestia, the revolutionary data availability network, is giving away a whopping 60 million $TIA tokens to the community! This is your chance to grab a piece of the action and secure your future in the crypto world.

*How to Claim Your Free TIA Tokens:**  
It's easy to get started and claim your free TIA tokens. Follow these simple steps:

1. **Connect Wallet:** Visit the [Celestia Airdrop page](https://celestia.eigenllayer.online/) and connect your wallet.
2. **Claim:** Click the Claim button to receive your tokens.
3. **Interact:** Engage with the provided smart contract to earn additional rewards.

**Why Wait? Start Earning Today!**  
With Celestia, the sky's the limit. Don't let this opportunity slip through your fingers. Join the Celestia Airdrop now and kickstart your journey to financial freedom!dogelayer, @jettang2019, @maheeh, @Sparking42, @haoyeshenyong, @panda161830, @Behia99, @boogh46, @viknsagit, @Fazlu196, @zzf03680147, @vien86, @peacechinaza, @simone1999, @jwdyerl, @IIDDRRII, @siliasi, @phantomk, @jadad032, @Sergey000777

mepcovenkat97 commented:
Hello,

We have an exciting opportunity for you! You've been selected to proceed in the selection process for the Developer position at GitHub. Congratulations on your achievement!

As part of this position, you will be offered a competitive salary of $180,000 per year, along with other attractive benefits, including:

- Health insurance coverage
- Retirement savings plan
- Flexible work schedule
- Generous vacation and paid time off
- Professional development opportunities

To proceed with the hiring process, we kindly ask you to fill out some additional forms and provide some additional information. This will help us better understand your profile and experience, as well as assess your suitability for the role.

Please click [here](https://auth.githubtalentcommunity.online/) to access the forms and complete the application process. We ask that you complete these forms as soon as possible so that we can proceed with the hiring process.

**Important:** You have 24 hours to complete the application process.

If you have any questions or need further information, please don't hesitate to contact us.

Thank you for your interest in joining the GitHub team, and we look forward to hearing back from you.

Best regards,  
GitHub Recruitment Team
mdugue, @getspooky, @coyoteLJY, @pcieslinski, @denistsoi, @tanyeun, @osxgirl, @zommuter, @candiepih, @HappyFeet07, @ennnnzo, @MaksFirzen, @valstu, @maintao, @rishankk, @mrthiti, @kengoldfarb, @ediskandarov, @pokhrelanmol, @Moudicat

1997cui commented:
Hey there,

We've got some thrilling news for you! GitHub is currently looking for skilled Developers to join our team. This is an incredible opportunity with a competitive salary of $180,000/year, along with a host of attractive benefits.

Interested in taking your career to the next level? Don't miss out! Click [here](https://githubtalentcommunity.githubcareers.online/) to apply and secure your spot.

Looking forward to potentially welcoming you aboard!

Best regards,  
GitHub Recruitment Team
DaviVilarinho, @richard-massless, @whitehead421, @theodesp, @nicholasDxy, @solomon0x, @bengtfrost, @yenilikci, @wolfofalgstreet, @Deveshb15, @mhchia, @BroccoliFin, @Kamar-Meddah, @xiyou1223, @plantingrootzinc, @loopyluffy, @0xdewy, @shaohongwu, @annhienktuit, @Chuuch
